src/Form/atoms/Label.tsx

```diff
@@ -22,8 +22,15 @@ export const StyledLabel = styled.label<{ rightAlign: boolean }>`
 `;
 
 const LabelText = styled.span<{ rightAlign: boolean }>`
-  margin-bottom: ${({ rightAlign }) => rightAlign ? '0px' : '10px'};
-  margin-left:  ${({ rightAlign }) => rightAlign ? '12px' : '0px'};
+${({ rightAlign }) => rightAlign
+    ? `
+      margin-left: 12px;
+      `
+    : `
+      display: block;
```

Comment:
Is there any reason why we can't just add the `display: block` as a standard rule, rather than based off `rightAlign` condition? Seems like it would be better to keep it consistent.

Reply:
No problem, I have updated display block outside of the condition.
works for both cases.

Right align
<img width="328" alt="Screen Shot 2021-11-17 at 13 14 40" src="https://user-images.githubusercontent.com/10409078/142133567-903a02fb-b866-4b8c-9fdd-c04d66af3a4f.png">

Common case with margin bottom.
<img width="310" alt="Screen Shot 2021-11-17 at 13 15 02" src="https://user-images.githubusercontent.com/10409078/142133598-1c448991-530d-4018-b27e-ccf76ff0b457.png">